sdk/managed/test/Microsoft.WindowsAzure.MobileServices.WindowsStore.Test/Functional/PushFunctional.Test.cs

```diff
@@ -31,6 +37,11 @@ public void InitialUnregisterAllAsync()
             push.UnregisterAllAsync(channelUri).Wait();
             registrations = push.ListRegistrationsAsync(channelUri).Result;
             Assert.IsFalse(registrations.Any(), "Deleting all registrations for a channel should ensure no registrations are returned by List");
+
+            channelUri = this.GetChannelUri().Result;
+            push.UnregisterAllAsync(channelUri).Wait();
```

Comment:
I'd prefer to do that actually, I just made minimal changes here to fit in with the existing code